pkg/ctl/cmdutils/cmd.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/weaveworks/eksctl/pkg/eks"
 )
 
+var Silent = false
```

Comment:
FWIW `var Silent bool` does result in false. :) But I understand that you wanted to be explicit. :)

Reply:
I'm not liking this global variable. :/

Reply:
nor do I, but its the cleanest way to handle the edge-case scenarios where `NewCtl` is called multiple times. I could change the func/have an arg but am I really going to change the 30+ places where this is called for the 1 edge case? I think this is about picking the lesser evils tbh

Reply:
The part that's happening more than once, can be used with a `sync.Once`. 

```go
    once.Do(func() {
        logRegionAndVersionInfo(c.ClusterConfig.Metadata)
    })
```

Or whatever it is you are guarding against here. This is global, and called only once, no matter how many times you call NewCtl.